lua/textcase/plugin/plugin.lua

```diff
@@ -135,17 +135,28 @@ end
 -- <cmd>h command-preview
 function M.incremental_substitute(opts, preview_ns, preview_buf)
   local command = "Subs"
+
+  -- preview_ns and preview_buf indicates the buffer to be modified
```

Comment:
This is clever

Reply:
This was there before from you ;) It just moved up a bit because I needed the buffer for my hack as well ;)